src/test/regress/sql/ddl_create.sql

```diff
@@ -0,0 +1,19 @@
+CREATE SCHEMA ddl_test_schema;
+SET search_path TO ddl_test_schema;
+
+CREATE TABLE test_tbl (a INT, b text, c BIGINT);
+SELECT create_distributed_table('test_tbl','a');
+
+CREATE INDEX CONCURRENTLY a_index ON test_tbl (a);
+
+BEGIN;
+    CREATE INDEX idx1 ON test_tbl (a) INCLUDE (b, c);
+    DROP TABLE test_tbl;
+ROLLBACK;
+
+ALTER TABLE test_tbl ADD CONSTRAINT cns UNIQUE (a);
+BEGIN;
+    ALTER TABLE test_tbl DROP CONSTRAINT cns;
+    INSERT INTO test_tbl VALUES (1,'a',1);
+    INSERT INTO test_tbl VALUES (1,'a',1);
+ROLLBACK;
```

Comment:
at the end of the test, we dropped all the indexes? Maybe keep as much as possible because we might add node or rebalance on these objects at some point, where existence of such indexes would be good to test